tests/integration/targets/scaleway_container_registry_info/tasks/main.yml

```
@@ -0,0 +1,46 @@
+---
+####################################################################
+# WARNING: These are designed specifically for Ansible tests       #
+# and should not be used as examples of how to write Ansible roles #
+####################################################################
+
+# Copyright (c) 2022, Guillaume MARTINEZ <lunik@tiwabbit.fr>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: Create container_registry
+  community.general.scaleway_container_registry:
+    state: present
+    name: '{{ name }}'
+    region: '{{ scaleway_region }}'
+    project_id: '{{ scw_project }}'
+    description: '{{ description }}'
+  register: integration_container_registry
+
+- name: Get container registry info
+  community.general.scaleway_container_registry_info:
+    name: '{{ name }}'
+    region: '{{ scaleway_region }}'
+    project_id: '{{ scw_project }}'
+  register: cr_info_task
+
+- ansible.builtin.debug: var=cr_info_task
```

Expressing the module parameters as keys in a YAML's dict is the preferred convention.
```suggestion
- ansible.builtin.debug:
    var: cr_info_task
```

---

Fixed in ac03fd5c5b30d99f508cb2699f9ff3513fabc44a